Formula/ccls.rb

```diff
@@ -0,0 +1,37 @@
+class Ccls < Formula
+  desc "C/C++ language server"
+  homepage "https://github.com/MaskRay/ccls"
+  url "https://github.com/MaskRay/ccls.git", :tag => "0.20180513", :revision => "19d0aad2ca9663b5633b575efe41803005b21d38"
+  head "https://github.com/MaskRay/ccls.git"
+
+  option "with-build-debug", "Configures ccls to be built in debug mode"
+  option "without-system-clang", "Downloading Clang from http://releases.llvm.org/ during the configure process"
+
+  depends_on "cmake" => :build
+  depends_on "llvm@6" => :build
+
+  def install
+    build_type = build.with?("build-debug") ? "debug" : "release"
+    system_clang = build.with?("system-clang") ? "ON" : "OFF"
+
+    args = std_cmake_args + %W[
+      -DSYSTEM_CLANG=#{system_clang}
+      -DCMAKE_BUILD_TYPE=#{build_type.capitalize}
```

Comment:
`CMAKE_BUILD_TYPE` is set by `std_cmake_args`

Reply:
Sometimes need to override CMAKE_BUILD_TYPE to switch build type ( Release / Debug).

Reply:
Not once the option is removed :)